README.md

```diff
@@ -83,20 +87,31 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get("/", function(req, res){
+  response.send("index");
+});
+app.post("/", function(req, res){
+  response.send("create");
+});
+app.put("/:id", function(req, res){
+  response.send("update");
+});
+app.delete("/:id", function(req, res){
+  response.send("delete");
+});
 ```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Rails is very formal and Express is more informal and flexible as far as where you can put certain files.
```

Comment:
👍 focus on using the words "opinions" and "rules" in your answer. Think about rails' convention over configuration